examples/use-camera-and-microphone-example/src/MainControls.tsx

```diff
@@ -61,6 +62,7 @@ const autostartAtom = atomWithStorage<boolean>("autostart", false, undefined, {
 
 export const MainControls = () => {
   const [token, setToken] = useAtom(tokenAtom);
+  useReconnectLogs();
```

Comment:
this example is weird, is it a testing app or an example for the user? Because users generally don't need to log reconnection attempts into the console...

Reply:
It's a testing app, but unfortunately it's located in the example directory. I've added a `// for debugging` comment to indicate that it's not necessary because I don't want to spend more time creating a whole new testing app.